src/include/module/audio/audio_stream.h

```diff
@@ -13,6 +13,11 @@
 #include <stdbool.h>
 #include "../ipc/stream.h"
 
+enum sof_audio_stream_state {
+	STREAM_STATE_INITIAL,			/* Initial state, hw params not configured. */
+	STREAM_STATE_READY			/* Stream ready, hw params configured */
```

Comment:
if you use a comma after `READY` you don't have to replace the line in the next commit